src/lib/planviz/ast_visualizer.cpp

```diff
@@ -13,28 +13,43 @@
 #include "optimizer/table_statistics.hpp"
 #include "sql/sql_query_plan.hpp"
 #include "utils/assert.hpp"
+#include "viz_utils.hpp"
 
 namespace opossum {
 
-void ASTVisualizer::visualize(const std::vector<std::shared_ptr<AbstractASTNode>>& ast_roots,
-                              const std::string& dot_filename, const std::string& img_filename) {
+ASTVisualizer::ASTVisualizer(const std::vector<std::shared_ptr<AbstractASTNode>>& _ast_roots,
```

Comment:
Code Style: Why do we have underscore in parameters?

Reply:
C&P error...